packages/core/test/test-util/test-services/v4/test-service/action-imports.ts

```diff
@@ -230,7 +230,7 @@ export function testActionImportSharedEntityReturnTypeCollection(
 ): Omit<
   ActionImportRequestBuilder<
     TestActionImportSharedEntityReturnTypeCollectionParameters,
-    never
+    never[]
```

Comment:
Hi, me again, about this `never[]` ;)
[`never`](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#the-never-type) represents a type that you cannot assign anything to, e. g. if you throw an error. What would `never[]` mean?

I think this is only used when we have no way of knowing what type to deserialize with `execute`. When deserializing by using `execute`, we throw an error, therefore the return value is `never` for `execute`.
When using `executeRaw` this type is not used.